spec/unit/models/thread.spec.ts

```diff
@@ -0,0 +1,105 @@
+import { MatrixEvent } from "../../../src/models/event";
```

Comment:
we should probably start introducing copyright to our tests instead of copy/pasting history, but that might also need to go to the team for general notice.

I'm happy either way for this PR, but definitely something we should deal with *eventually*.